caffe2/contrib/CMakeLists.txt

```diff
@@ -32,5 +32,7 @@ set(Caffe2_GPU_BINARY_SRCS ${Caffe2_GPU_BINARY_SRCS} PARENT_SCOPE)
 
 # HIP sources, include, test sources
 set(Caffe2_HIP_SRCS ${Caffe2_HIP_SRCS} PARENT_SCOPE)
-set(Caffe2_HIP_TEST_SRCS ${Caffe2_HIP_TEST_SRCS} PARENT_SCOPE)
 set(Caffe2_HIP_INCLUDE ${Caffe2_HIP_INCLUDE} PARENT_SCOPE)
+set(Caffe2_HIP_DEPENDENCY_LIBS ${Caffe2_HIP_DEPENDENCY_LIBS} PARENT_SCOPE)
+set(Caffe2_HIP_TEST_SRCS ${Caffe2_HIP_TEST_SRCS} PARENT_SCOPE)
+set(Caffe2_HIP_BINARY_SRCS ${Caffe2_HIP_BINARY_SRCS} PARENT_SCOPE)
```

Comment:
Is this variable used somewhere?

Reply:
I meant `Caffe2_HIP_BINARY_SRCS`

Reply:
Not entirely sure.  Added here for parity with the GPU/CUDA statements from above in the same file.

Reply:
Removed Caffe2_HIP_BINARY_SRCS var.